ts/navigation/AppNavigator.ts

```diff
@@ -36,6 +38,9 @@ const navigator = createStackNavigator(
     },
     [ROUTES.WORKUNIT_GENERIC_FAILURE]: {
       screen: WorkunitGenericFailure
+    },
+    [ZENDESK_ROUTES.HELP_CENTER]: {
```

Comment:
we should add these screens only when the FF is enabled
see https://github.com/pagopa/io-app/blob/1c2d2b54b96ad5b075e702857f8517b1ea06759d/ts/navigation/WalletNavigator.ts#L222

Reply:
Thank you, added in [51f6383](https://github.com/pagopa/io-app/pull/3538/commits/51f6383303aa5c25d656b9df10a89967a94cf997)